### PR TITLE
Move type comment to type annotation

### PR DESCRIPTION
**Pull request by eggplants**
Changed the style of type hinting from type comments into annotations.

Example:

```python
def foo():
    # type: () -> None
    return None
```

↓

```python
def foo() -> None:
    return None
```

### CODE REVIEW

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -8,8 +8,9 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
       matrix:
-        python-version: ['3.6', '3.10']
```
Comment (Xdynix) [inline .github/workflows/pythonapp.yml:12]: Some concern: It has been observed before that there are too many Github Action tests in a short period of time, resulting in the call frequency of `api.auth()` exceeding the limit. Adding more versions may make automated tests more unstable and require manual retry.

**Comment by eggplants** [comment]
I don't know the exact reason, but if you make this change, up to 3.8 raises following error:

```
Traceback (most recent call last):
  File "demo.py", line [7](https://github.com/upbit/pixivpy/runs/5297620093?check_suite_focus=true#step:7:7), in <module>
    from pixivpy3 import AppPixivAPI
  File "/home/runner/work/pixivpy/pixivpy/pixivpy3/__init__.py", line 6, in <module>
    from .aapi import AppPixivAPI
  File "/home/runner/work/pixivpy/pixivpy/pixivpy3/aapi.py", line [8](https://github.com/upbit/pixivpy/runs/5297620093?check_suite_focus=true#step:7:8), in <module>
    from .api import BasePixivAPI
  File "/home/runner/work/pixivpy/pixivpy/pixivpy3/api.py", line 12, in <module>
    from .utils import JsonDict, ParamDict, ParsedJson, PixivError, Response
  File "/home/runner/work/pixivpy/pixivpy/pixivpy3/utils.py", line 12, in <module>
    class PixivError(Exception):
  File "/home/runner/work/pixivpy/pixivpy/pixivpy3/utils.py", line 1[9](https://github.com/upbit/pixivpy/runs/5297620093?check_suite_focus=true#step:7:9), in PixivError
    body: Optional[str] = None,
TypeError: 'ABCMeta' object is not subscriptable
```

**Comment by eggplants** [comment]
It seems to be related to the fact that type comment can use a syntax that is several versions ahead.

e.g.) In 3.8, `# type: (int|str) -> None` is valid, but `def (v: int|str) -> None: ...` is not.

**Comment by eggplants** [comment]
Is there a syntax included in types-requests that can be used directly in 3.9 and above?

**Comment by eggplants** [comment]
It looks like we can leave Type comment for now.

**Comment by eggplants** [comment]
`CaseInsensitiveDict[Any]` is maybe the culprit of this problem...

**Comment by eggplants** [comment]
CI failed, but I couldn't understand where I have to fix from its log. `TypeError: 'ABCMeta' object is not subscriptable` is maybe fixed. 

https://github.com/upbit/pixivpy/runs/5353329803?check_suite_focus=true

**Review by Xdynix — APPROVED** [review]

**Comment by Xdynix** [comment]
Thanks for contribution! Hold for @upbit's review.

**Review by Xdynix — APPROVED** [review]

**Comment by eggplants** [comment]
Please squash commits when merging.

File: .github/workflows/pythonapp.yml (hunk 1 above)
Comment (eggplants) [reply .github/workflows/pythonapp.yml:12]: 9138a40054856cfa6612b72e4b071e321b76a77e

**Comment by upbit** [comment]
Sorry for late review, merged. Forgot to squash commits, 下次一定 🤓

**Pull request merged**